src/utils/tasks.js

```diff
@@ -105,3 +105,21 @@ export {
   latestJob,
   jobMessageOutputs
 }
+
+export function dtMean (taskNode) {
+  // Use luxon.Duration to parse the mean as milliseconds,
+  // then convert to a nice iso formatted duration:
+  const dur = taskNode.node?.task?.meanElapsedTime
+  if (dur) {
+    const seconds = dur % 60
+    const minutes = (dur - seconds) % 60
+    const hours = dur - minutes - seconds
+    const ret = hours.toString().padStart(2, '0') +
```

Comment:
I haven't tested, but does this work OK if hours > 99?  And relatedly, tasks (e.g. climate models) could potentially run for more 24 hours, so should we use days too if necessary?

Reply:
There's a whole conversation in the resolved comment about not using an external dependency where I said

> it's a local assumption that Cylc tasks won't be > 24 hours long.

However, this is a seriously minority case. I will investigate what happens if hours > 99.

I do think that days is probably unnecessary though.

Reply:
How about:

```bash
[DDd] HH:MM:SS

01:02:03     # PT1H2M3S
2d 01:02:03  # P2DT1H2M3S
```

Reply:
Seems like a legit upgrade. I'll bite.